files/fr/web/css/@property/index.html

```diff
@@ -0,0 +1,80 @@
+---
+title: '@property'
+slug: Web/CSS/@property
+browser-compat: css.at-rules.property
+translation_of: 'Web/CSS/@property'
+---
+<div>{{CSSRef}}{{SeeCompatTable}}</div>
+
+<p>La règle {{cssxref("at-rule")}} <a href="/en-US/docs/Web/CSS">CSS</a> <strong><code>@property</code></strong> fait partie du panel d'API <a href="/fr/docs/Web/Houdini">CSS Houdini</a>, qui permet aux développeurs de définir explicitement leurs propriétés CSS personnalisées (voir {{cssxref('--*', 'CSS custom properties')}}). Cela permet la vérification des types de propriétés, de leur donner des valeurs par défaut et de déterminer si une propriété peut hériter ses valeurs ou non.</p>
```

Comment:
Link to target /fr/ and not /en-US/ (comment for all the links on this page)

Reply:
Sans vouloir verser dans le débat de l'écriture inclusive, on peut retirer "aux développeurs" pour garder une formulation épicène (sans alourdir la phrase).

Reply:
@SphinxKnight alors en fait… je suis un peu gêné justement de voir beaucoup de termes masculins (genre "utilisateur", présent partout) alors que je préférerais une formulation épicène ou la combinaison des termes masculins et féminins. 

Je prévoyais de remonter cela dans une issue une fois que j'aurais un peu plus d'expérience sur le reste de la contribution à MDN.

Pour l'heure je vais faire cette petite correction, mais je pense que c'est un débat important pour l'ensemble des ressources :)

Reply:
Tout à fait d'accord pour ces modifications, dans d'autres cas j'aurai privilégié "développeuses et développeurs" par exemple.
Je pense qu'on peut tout à fait suivre les règles générales indiquées avec https://github.com/mozfr/besogne/wiki/Guide-stylistique-pour-la-traduction#sadresser-%C3%A0-lutilisateur-ou-lutilisatrice (document qui s'adresse plus largement aux activités de traduction francophones pour Mozilla)

Reply:
fixed in https://github.com/mdn/translated-content/pull/1762/commits/67d49a69d34fe3a228089c4691091a4839f0c8e8